Doc/whatsnew/3.13.rst

```diff
@@ -191,6 +191,14 @@ ipaddress
 * Add the :attr:`ipaddress.IPv4Address.ipv6_mapped` property, which returns the IPv4-mapped IPv6 address.
   (Contributed by Charles Machalow in :gh:`109466`.)
 
+mmap
+----
+
+* The :class:`mmap.mmap` class now has an :meth:`~mmap.mmap.seekable` method
+  that can be used where it requires a file-like object with seekable and
+  the :meth:`~mmap.mmap.seek` method return the current position.
```

Comment:
```suggestion
  the :meth:`~mmap.mmap.seek` method return the new absolute position.
```

See: https://docs.python.org/dev/library/io.html#io.TextIOBase.seek

By the way, it would be nice to complete https://docs.python.org/dev/library/io.html#io.IOBase.seek doc to say that the function returns the new absolute position.